js/files/circles.files.list.js

```diff
@@ -124,11 +124,20 @@
 					},
 
 					formatResult: function(circle) {
-						return circle.name;
+						return this.escapeHTML(circle.name);
 					},
 
 					formatSelection: function(circle) {
-						return circle.name;
+						return this.escapeHTML(circle.name);
+					},
+
+					escapeHTML: function (text) {
+						return text.toString()
+							.split('&').join('&amp;')
+							.split('<').join('&lt;')
+							.split('>').join('&gt;')
+							.split('"').join('&quot;')
+							.split('\'').join('&#039;');
```

Comment:
```suggestion
  						const parser = new DOMParser();
  						const doc = parser.parseFromString(text, 'text/html');
  						return doc.body.textContent;
```

Reply:
This would escape any chars not thought about and handle more edge cases, it however looks okay